.eslintrc.js

```diff
@@ -23,7 +23,6 @@ module.exports = {
     'no-unneeded-ternary': 'off',
     'no-unused-expressions': 'off',
     'no-unused-vars': 'off',
-    'node/no-deprecated-api': 'off',
```

Comment:
Non-strict assertions happen to be the only thing this linting rule currently fails on.